app/controllers/api/authentications_controller.rb

```diff
@@ -1,4 +1,11 @@
 module Api
   class AuthenticationsController < BaseController
+    def delete_resource(type, id, _data = {})
+      auth = resource_search(id, type, collection_class(:authentications))
+      task_id = auth.delete_in_provider_queue
+      action_result(true, "Deleting Authentication with id #{id}", :task_id => task_id)
+    rescue => err
+      raise "Could not delete authentication - #{err}"
```

Comment:
Let's also return action_result(false, ...) for rescue case, consistency otherwise our bulk actions won't play nice.  

Also, maybe update message to be consistent with others, maybe  "Deleting Authentication id:#{id}"  see helper methods provider_ident(), vm_ident(), maybe have something similar for authentications.